cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ListenersValidator.java

```diff
@@ -345,6 +347,19 @@ private static void validateBootstrapNodePort(Set<String> errors, GenericKafkaLi
             errors.add("listener " + listener.getName() + " cannot configure bootstrap.nodePort because it is not NodePort based listener");
         }
     }
+    
+    /**
+     * Validates that bootstrap.nodePort is used only with NodePort type listener
+     *
+     * @param errors    List where any found errors will be added
+     * @param listener  Listener which needs to be validated
+     */
+    private static void validateBootstrapExternalIPs(Set<String> errors, GenericKafkaListener listener) {
```

Comment:
You should write unit tests (or add it to one of the existing unit tests I guess) for this method as well as the other one you added.